components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/GatewayCmdUtils.java

```diff
@@ -316,30 +315,55 @@ public static void createProjectStructure(String projectName) {
      * Create API-Files Directory for a particular project
      * @param projectName name of the project
      * @param apiId md5 hash value for apiName:version
-     * @param apiDefinition swagger content as a JSON string
      */
-    public static void createPerAPIFolderStructure(String projectName, String apiId, String apiDefinition){
-        String projectDir = getUserDir() + File.separator + projectName;
-
-        String apiFilesDirPath = projectDir + File.separator + GatewayCliConstants.PROJECTS_API_FILES_DIRECTORY_NAME;
+    public static void createPerAPIFolderStructure(String projectName, String apiId){
 
+        String apiFilesDirPath = getProjectAPIFilesDirectoryPath(projectName);
         String apiDirPath = apiFilesDirPath + File.separator + apiId;
         createFolderIfNotExist(apiDirPath);
-        createFileIfNotExist(apiDirPath, GatewayCliConstants.API_METADATA_FILE);
+    }
 
+    public static void saveSwaggerDefinition(String projectName, String apiId, String apiDefinition){
         if (apiDefinition.isEmpty()){
             throw new CLIInternalException("No swagger definition is provided to generate API");
         }
-
         try {
-            writeContent(apiDefinition, new File (apiDirPath + File.separator +
-                    GatewayCliConstants.API_SWAGGER));
+            writeContent(apiDefinition, new File (GatewayCmdUtils.getProjectSwaggerFilePath(projectName, apiId)));
         } catch (IOException e) {
             throw new CLIInternalException("Error while copying the swagger to the project directory");
         }
+    }
+
+    private static void saveSwaggerDefinitionForSingleAPI(String projectName, ExtendedAPI api){
+        String swaggerString = SwaggerUtils.generateSwaggerString(api);
+        String apiId = HashUtils.generateAPIId( api.getName(), api.getVersion());
+        GatewayCmdUtils.saveSwaggerDefinition(projectName, apiId, swaggerString);
+    }
+
+    /**
+     * Save swagger definition for multiple APIs
```

Comment:
Need full stop and empty new line after the description 
```suggestion
     * Save swagger definition for multiple APIs.
     
```